index.js

```diff
@@ -0,0 +1,27 @@
+const { Command } = require('commander');
+const program = new Command();
+const version = require('./package.json').version;
+
+const launchBroker = require('./bin/broker/index');
+
+program
+    .name('newman-dashboard')
+    .addHelpCommand(false)
+    .version(version, '-v, --version');
+
+program
+    .option(
+        '-p, --port <port>',
+        'Specify the port to launch the dashboard',
+        '3000'
+    )
+    .action(() => {
+        if (process.env.NODE_ENV !== 'test') {
```

Comment:
Maybe we can have an option like `--daemonize`? Prevents all these checks for test environment + solve the `killall` problem.

Reply:
Done! ✨